tests/integration/targets/dcnm_network/tests/dcnm/overridden.yaml

```diff
@@ -143,6 +143,165 @@
+    - net_name: ansible-net12
+      vrf_name: Tenant-2
+      net_id: 7002
+      net_template: Default_Network_Universal
+      net_extension_template: Default_Network_Extension_Universal
+      vlan_id: 1511
+      gw_ip_subnet: '192.168.40.1/24'
+      mtu_l3intf: 8600
+      int_desc: 'test interface'
+      vlan_name: testvlan1
+      dhcp_srvr1_ip: '1.1.1.1'
+      dhcp_srvr2_ip: '2.2.2.2'
+      dhcp_srvr3_ip: '3.3.3.3'
+      dhcp_srvr1_vrf: one
+      dhcp_srvr2_vrf: two
+      dhcp_srvr3_vrf: three
+      attach:
+      - ip_address: "{{ ansible_switch2 }}"
+        ports: ["{{ ansible_sw2_int5 }}", "{{ ansible_sw2_int6 }}"]
+        deploy: true
+      deploy: false
+  register: result
+
+- assert:
+    that:
+    - 'result.changed == true'
+    - 'result.response[0].RETURN_CODE == 200'
+    - 'result.response[1].RETURN_CODE == 200'
+    - 'result.response[2].RETURN_CODE == 200'
+    - 'result.response[3].RETURN_CODE == 200'
+    - '(result.response[2].DATA|dict2items)[0].value == "SUCCESS"'
+    - '(result.response[2].DATA|dict2items)[1].value == "SUCCESS"'
+    - 'result.diff[0].attach[0].deploy == true'
+    - 'result.diff[0].attach[1].deploy == true'
+    - '"{{ ansible_switch1 }}" in result.diff[0].attach[0].ip_address'
+    - '"{{ ansible_switch2 }}" in result.diff[0].attach[1].ip_address'
+    - 'result.diff[0].net_name == "ansible-net13"'
+    - 'result.diff[0].net_id == 7005'
+    - 'result.diff[0].vrf_name == "NA"'
+    - 'result.diff[0].is_l2only == true'
+    - 'result.diff[0].vlan_name == "testvlan"'
+    - 'result.diff[1].attach[0].deploy == true'
+    - '"{{ ansible_switch2 }}" in result.diff[1].attach[0].ip_address'
+    - 'result.diff[1].net_name == "ansible-net12"'
+    - 'result.diff[1].net_id == 7002'
+    - 'result.diff[1].vrf_name == "Tenant-2"'
+    - 'result.diff[1].is_l2only == false'
+    - 'result.diff[1].vlan_name == "testvlan1"'
+
+- name: OVERRIDDEN - sleep for 40 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 40
+
+- name: OVERRIDDEN - Override L2, L3 Networks with a new L2 network 
+  cisco.dcnm.dcnm_network: &conf2
+    fabric: "{{ ansible_it_fabric }}"
+    state: overridden
+    config:
+    - net_name: ansible-net14
+      vrf_name: NA
+      net_id: 7005
+      net_template: Default_Network_Universal
+      net_extension_template: Default_Network_Extension_Universal
+      vlan_id: 1500
+      gw_ip_subnet: '192.168.30.1/24'
+      mtu_l3intf: 7600
+      int_desc: 'test interface'
+      vlan_name: testvlan
+      is_l2only: True
+      dhcp_srvr1_ip: '1.1.1.1'
+      dhcp_srvr2_ip: '2.2.2.2'
+      dhcp_srvr3_ip: '3.3.3.3'
+      dhcp_srvr1_vrf: one
+      dhcp_srvr2_vrf: two
+      dhcp_srvr3_vrf: three
+      attach:
+      - ip_address: "{{ ansible_switch2 }}"
+        ports: ["{{ ansible_sw2_int5 }}", "{{ ansible_sw2_int6 }}"]
+        deploy: true
+      deploy: false
+  register: result
+
+- assert:
+    that:
+    - 'result.changed == true'
```

Comment:
After the task above runs the only network that should remain is `ansible-net14` right?  Can we check the count of total networks here to make sure it's only 1?

Reply:
Added a check for network count and network name